src/main/java/com/cglx/Calculator.java

```diff
@@ -1,5 +1,11 @@
 package com.cglx;
-
+import java.io.BufferedReader;
+import java.io.File;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.File'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AUselessImportCheck)